models/Profile.js

```diff
@@ -0,0 +1,73 @@
+const mongoose = require('mongoose')
+
+const ProfileSchema = new mongoose.Schema({
+  avatar: {
+    type: String
+  },
+  website: {
+    type: String
+  },
+  location: {
+    type: String
+  },
+  status: {
+    type: String,
+    required: true
+  },
+  bio: {
+    type: String
+  },
+  place: [
+    {
+      title: {
+        type: String,
+        required: true
+      },
+      description: {
+        type: String
+      },
+      location: {
+        type: String,
+        required: true
+      },
+      image: {
+        type: String,        
+      },
+      coordinates: {
+        lat: {type: Number, required: true},
+        lng: {type: Number, required: true}
+      },
+      date: {
+        type: Date,
+        default: Date.now
+      },
+    }
+  ],
+  social: {
+    youtube: {
+      type: String
+    },
+    twitter: {
+      type: String
+    },
+    facebook: {
+      type: String
+    },
+    linkedin: {
+      type: String
+    },
+    instagram: {
+      type: String
+    }
+  },
+  date: {
+    type: Date,
+    default: Date.now
+  },
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
```

Comment:
It will get too crowded in one schema file. I'll be clearer and easier to develop the app in two schemas in my opinion.

Reply:
Will it have performance impact?

Reply:
I could have a performance impact. The app will have a Post model and this will also be linked with the User model, it will give more clarity and it will be easier to visualize the app by parts.